DigitalPuzzle/DigitalPuzzle.ino

```diff
@@ -1,349 +1,238 @@
-      return IDStorage;
-    }
-    delay(100);
-    rfid.halt(); //command the card into sleep mode 
-}
-
-
-/*************************************************************
- * Function：GetReaderResult
- * Description：read the UID of the given tag 
- * Input parameter：number of the reader that needs to be read
- * Return：returns boolean; the result of the placed logic gate
- *************************************************************/
-bool *GetReaderResult(int ReaderSelect)
-{
-  switch(ReaderSelect)
-  {
-    case 0:
-      switch(CompareNFC(ReadRFID(0)))
-      {
-        case 1:   //AND GATE
-          ReaderResult[0] = Check_AND(StartValues[0], StartValues[1]);
-          break;
-        case 2:   //OR GATE
-          ReaderResult[0] = Check_OR(StartValues[0], StartValues[1]);
-          break;
-        case 3:   //NAND GATE
-          ReaderResult[0] = Check_NAND(StartValues[0], StartValues[1]);
-          break;
-        case 4:   //NOR GATE
-          ReaderResult[0] = Check_NOR(StartValues[0], StartValues[1]);
-          break;
-        case 5:   //XOR GATE
-          ReaderResult[0] = Check_XOR(StartValues[0], StartValues[1]);
-          break;
-        default:  //
-          ReaderResult[0] = 0; 
-          break;
-      }
-    case 1:
-      switch(CompareNFC(ReadRFID(1)))
-      {
-        case 1:   //AND GATE
-          ReaderResult[1] = Check_AND(StartValues[2], StartValues[3]);
-          break;
-        case 2:   //OR GATE
-          ReaderResult[1] = Check_OR(StartValues[2], StartValues[3]);
-          break;
-        case 3:   //NAND GATE
-          ReaderResult[1] = Check_NAND(StartValues[2], StartValues[3]);
-          break;
-        case 4:   //NOR GATE
-          ReaderResult[1] = Check_NOR(StartValues[2], StartValues[3]);
-          break;
-        case 5:   //XOR GATE
-          ReaderResult[1] = Check_XOR(StartValues[2], StartValues[3]);
-          break;
-        default:  //
-          ReaderResult[1] = 0; 
-          break;
-      }
-  }
-
-  return ReaderResult;
-}
-
-/*************************************************************
- * Function：ApplyResults
- * Description：turns the ledstrips on and off, depending on
-   the result. 
- * Input parameter：none
- * Return：returns none
- *************************************************************/
-void ApplyResults()
-{
-  if (GetReaderResult(0)[0] == true)
-  {
-    digitalWrite(GreenPins[0], HIGH);
-    digitalWrite(RedPins[0], LOW);
-  }
-  else
-  {
-    digitalWrite(GreenPins[0], LOW);
-    digitalWrite(RedPins[0], HIGH);
-  }
-
-  if (GetReaderResult(1)[1] == true)
-  {
-      digitalWrite(GreenPins[1], HIGH);
-      digitalWrite(RedPins[1], LOW);
-  }
-  else
-  {   digitalWrite(GreenPins[1], LOW);
-      digitalWrite(RedPins[1], HIGH);
-  }
+	switch (ReaderSelect) //Select one of the eight readers 
+	{                     //Pin Configuration of RFID.begin (IRQ_PIN,SCK_PIN,MOSI_PIN,MISO_PIN,SDA_PIN,RST_PIN)
+	case 0:
+		rfid.begin(12, 13, 9, 24, 10, 11);
```

Comment:
Allemaal deze pinnen moeten even in een variabele gestopt worden zodat je ze gemakkelijk in 1 keer kan veranderen voor **ALLE** readers

Reply:
Zeer goede suggestie